tensorflow/core/kernels/data/experimental/io_ops.cc

```diff
@@ -148,7 +148,8 @@ Status SaveDatasetOp::GetShardIndex(IteratorContext* ctx,
   }
   std::vector<Tensor> output_tensors;
   TF_RETURN_IF_ERROR(
-      function->RunWithBorrowedArgs(ctx, element, &output_tensors));
+      function->RunWithBorrowedArgs(ctx, element, &output_tensors,
+        std::shared_ptr<model::Node>(nullptr)));
```

Comment:
Does passing simply `nullptr` not work? To improve readability, use the following style:

`function->RunWithBorrowedArgs(ctx, element, &output_tensors, /*node=*/nullptr);`